docs/native/client-side-api.md

```diff
@@ -0,0 +1,125 @@
+# Client Side API
+
+The functions are available on the client side in JavaScript to allow extending the default behaviour of the feature beyond recording page views.
+
+**`Altis.Analytics.onReady( callback <function> )`**
+
+Use this function to ensure analytics has loaded before making calls to `Altis.Analytics.registerAttribute()` or `Altis.Analytics.record()` for example.
+
+## Endpoint Functions
+
+**`Altis.Analytics.updateEndpoint( data <object> )`**
+
+Updates the data associated with the current user. Use this to provide updated custom user attributes and metrics, a user ID, and demographic data.
+
+The endpoint data is merged with all new event records associated with that endpoint.
+
+The accepted data shape is as follows:
+
+```json
+{
+  "Address": "string",
+  "Attributes": { "string": [ "string", ... ] ... },
+  "Demographic": {
+    "AppVersion": "string",
+    "Locale": "string",
+    "Make": "string",
+    "Model": "string",
+    "ModelVersion": "string",
+    "Platform": "string",
+    "PlatformVersion": "string",
+    "Timezone": "string"
+  },
+  "Location": {
+    "City": "string",
+    "Country": "string",
+    "Latitude": double,
```

Comment:
We should avoid mixing these field descriptions; either `"string", "double"` as valid JSON or `string, double` as pseudo-TypeScript.

Reply:
hm yeah, the number wouldn't be quoted, not sure how best to communicate this otherwise. It's copied from the previous docs

Reply:
Just unquote it all then, `"City": string` is fine